components/form-builder/app/navigation/EditNavigation.tsx

```diff
@@ -20,32 +20,51 @@ export const EditNavigation = () => {
   };
 
   return (
-    <div className="relative">
```

Comment:
Because the design does not fit a traditional component like a Switch or Radio or.. the below required some umm.. creativity. The main problem this solves is having the button state announce when it changes. 
This could be moved into a component in the future but I tried to change as little as possible.